### PR TITLE
AC Stub Test - LRAC-8826, LRAC-8840 | master

### PR DESCRIPTION
**Pull request by AfonsoHSantos**
https://issues.liferay.com/browse/LRAC-8826
https://issues.liferay.com/browse/LRAC-8840

### CODE REVIEW

**Comment by AfonsoHSantos** [comment]
ci:test:sf

**Comment by AfonsoHSantos** [comment]
ci:test:sf

**Comment by AfonsoHSantos** [comment]
ci:test:sf

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/AnalyticsCloudConnectionTest.testcase [hunk 1]
```diff
@@ -115,21 +169,93 @@ definition {
 		var propertyName = ACDXPSettings.connectDXPtoAnalyticsCloud();
 	}
 
-	@description = "Automation ID: LRAC-8128 | Test Summary: View asset added by DXP data source"
+	@description = "Automation ID: LRAC-8821 | Test Summary: Data Source list is searchable"
+	@ignore = "true"
+	@priority = "4"
+	test DataSourceListIsSearchable {
+
+		// TODO LRAC-8821 DataSourceListIsSearchable
```
Comment (alexcandidos) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/AnalyticsCloudConnectionTest.testcase:177]: hi @AfonsoHSantos , it should be a P5.

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/AnalyticsCloudConnectionTest.testcase [hunk 2]
```diff
@@ -115,21 +169,93 @@ definition {
...
+
+	@description = "Automation ID: LRAC-8822 | Test Summary: Sync Liferay Data Source Contacts by User Groups"
+	@ignore = "true"
+	@priority = "5"
+	test SyncDataSourceContactsByGroups {
+
+		// TODO LRAC-8822 SyncDataSourceContactsByGroups
+
+	}
+
```
Comment (alexcandidos) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/AnalyticsCloudConnectionTest.testcase:243]: hi @AfonsoHSantos , it should be a P4.

**Review by alexcandidos — CHANGES_REQUESTED** [review]
hi @AfonsoHSantos , could you please do the fixes?
Thanks!

**Comment by AfonsoHSantos** [comment]
ci:test:sf

**Comment by AfonsoHSantos** [comment]
ci:test:sf

**Comment by AfonsoHSantos** [comment]
ci:test:sf

**Pull request closed without merging**